wasm/jit/jit_amd64.go

```diff
@@ -1312,31 +1312,131 @@ func (c *amd64Compiler) performDivisionOnInts(is32Bit bool, signed bool) error {
 // on the top two values of float type on the stack, placing the result back onto the stack.
 // For example, stack [..., 1.0, 4.0] results in [..., 0.25].
 func (c *amd64Compiler) compileDivForFloats(is32Bit bool) error {
+	if is32Bit {
+		return c.emitSimpleBinaryOp(x86.ADIVSS)
+	} else {
+		return c.emitSimpleBinaryOp(x86.ADIVSD)
+	}
+}
+
+// compileShr emits instructions to perform and operation on
+// top two values on the stack, and pushes the result.
+func (c *amd64Compiler) compileAnd(o *wazeroir.OperationAnd) (err error) {
+	switch o.Type {
+	case wazeroir.UnsignedInt32:
+		err = c.emitSimpleBinaryOp(x86.AANDL)
+	case wazeroir.UnsignedInt64:
+		err = c.emitSimpleBinaryOp(x86.AANDQ)
+	}
+	return
+}
+
+// compileShr emits instructions to perform or operation on
```

Comment:
because or is also English it makes the sentence weird unless we capitalize or quote it

```suggestion
// compileOr emits instructions to perform an "or" operation on
```